src/main/java/io/opensaber/views/EvaluatorFactory.java

```diff
@@ -0,0 +1,34 @@
+package io.opensaber.views;
+
+public class EvaluatorFactory {
+    
+    public static final String concat = "concat";
```

Comment:
MANDATORY: Why are these hardcoded here? We are creating a library and this can't be in a Factory class for sure.